analytics/tests/logs/test_logging.py

```diff
@@ -40,7 +40,7 @@ def test_init(
         records = caplog.records
         assert len(records) == 2
         assert re.match(
-            r"^start test_logging: \w+ [0-9.]+ \w+, hostname \S+, pid \d+, user \d+\(\w+\)$",
+            r"^start test_logging: \w+ [0-9.]+ \w+, hostname \S+, pid \d+, user \d+\([\w\.]+\)",
```

Comment:
Changed this because the tests were failing locally if there was a period in the username, e.g. `billy.daly`